api/src/models/language.js

```diff
@@ -0,0 +1,43 @@
+const mongoose = require('mongoose');
+
```

Comment:
One more addition to this file that I forgot to include in the issue. We will need to set some of the fields, like _admin_id, as single or compound indexes on the schema level. 

Course and Course Details don't need indexes.
Unit will need a compound index on `_course_id` and order to _order. Both can have a sort order of 1
Lesson will need a compound index `on_unit_id`, `_course_id`, and `_order.` All three can have a sort order of 1.
Vocab will need a single index on `_order`. It can have a sort order of 1.


Refer to the docs below on how to do all of this:

Documentation: 
- https://mongoosejs.com/docs/guide.html#indexes (has the code)
- https://docs.mongodb.com/manual/indexes/ (more detailed explanation)